<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayingWindow</class>
 <widget class="QMainWindow" name="PlayingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1722</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>135</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>270</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>250</number>
        </property>
        <property name="rightMargin">
         <number>400</number>
        </property>
        <item>
         <widget class="QFrame" name="frame">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 127);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>40</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="Tcard_1">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_2">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_3">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_4">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_5">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="Tcard_6">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_7">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_8">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_9">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="Tcard_10">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>250</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>40</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="Pcard_4">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>135</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>270</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_1">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>135</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 0, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>135</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 0, 255);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>135</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(170, 0, 255);
</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pcard_5">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>135</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>270</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>200</number>
        </property>
        <item>
         <widget class="QLabel" name="Cscore">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 255);
border-radius: 50%;
color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Computer

Score

0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Pscore">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(170, 0, 255);
border-radius: 50%;
color: rgb(255, 255, 255);
font: 14pt &quot;Segoe UI&quot;;
</string>
          </property>
          <property name="text">
           <string>Player Score

0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1722</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
